packages/base/src/stacBrowser/StacBrowser.tsx

```diff
@@ -1,37 +1,13 @@
-import {
-  IJupyterGISModel,
-  IJupyterGISTracker,
-  IJupyterGISWidget,
-} from '@jupytergis/schema';
-import React, { useEffect, useState } from 'react';
+import { IJupyterGISModel } from '@jupytergis/schema';
+import * as React from 'react';
 
 import StacPanelView from '@/src/stacBrowser/components/StacPanelView';
-import { IControlPanelModel } from '@/src/types';
 
 interface IStacBrowserDialogProps {
-  controlPanelModel: IControlPanelModel;
+  model: IJupyterGISModel;
 }
 
-const StacBrowser = ({ controlPanelModel }: IStacBrowserDialogProps) => {
-  const [jgisModel, setJgisModel] = useState<IJupyterGISModel | undefined>(
-    controlPanelModel?.jGISModel,
-  );
-
-  useEffect(() => {
-    const handleCurrentChanged = (
-      _: IJupyterGISTracker,
-      widget: IJupyterGISWidget | null,
-    ) => {
-      setJgisModel(widget?.model);
-    };
-
-    controlPanelModel.documentChanged.connect(handleCurrentChanged);
-
-    return () => {
-      controlPanelModel.documentChanged.disconnect(handleCurrentChanged);
-    };
-  }, [controlPanelModel]);
-
-  return <StacPanelView model={jgisModel} />;
+const StacBrowser: React.FC<IStacBrowserDialogProps> = ({ model }) => {
+  return <StacPanelView model={model} />;
```

Comment:
I should have noticed this before, sorry :grimacing: Now that this component is cleaned up, it's not really doing anything. We could move the contents of the `StacPanelView` component here and delete `StacPanelView.tsx`. Or we could get rid of the `StacBrowser` component and directly use `StacPanelView` (but I think `StacBrowser` is a better name, personally). What do you think?